internal/js/modules/k6/browser/browser/page_mapping.go

```diff
@@ -926,6 +945,9 @@ func waitForURLBodyImpl(vu moduleVU, target interface {
 	WaitForURL(urlPattern string, opts *common.FrameWaitForURLOptions, jsRegexChecker common.JSRegexChecker) error
 }, url sobek.Value, opts sobek.Value,
 ) (*sobek.Promise, error) {
+	if k6common.IsNullish(url) {
```

Comment:
Needs it in `frame.waitForUrl` too.

Reply:
This is being called by both. and the getBy* APIs are not on the frame